docs/guides/best-practices.md

```diff
@@ -110,3 +110,21 @@ app.route('/books', books)
 
 export default app
 ```
+The above code would work fine, however doing this would lose the typesafe
```

Comment:
Can you add the following sentence or a similar one?

> "If you want to use [the RPC feature](https://hono.dev/docs/guides/rpc)"

Reply:
of course 
if you want to use `RPC` features a better solution would be chaining the methods as described below